style.css

```diff
@@ -437,7 +437,7 @@
     background: #fff;
     border: 1px solid #cfd8dc;
     border-radius: 8px;
-    padding: 15px 38px 15px 0;
```

Comment:
### CONTACT SECTION

- Hey, the padding left and right is 24px. please, kindly update this part of your style. `#contact-me-section`


*your design*
![Screenshot from 2023-05-02 10-34-08](https://user-images.githubusercontent.com/59397159/235632286-1fcc181a-5780-4b12-98ad-491246f7c87e.png)

*Figma design*
![image](https://user-images.githubusercontent.com/59397159/235632409-eeaadc90-72ff-4a99-94cd-344ef3aba568.png)